conf/virtwho.yaml.template

```
@@ -0,0 +1,23 @@
+VIRTWHO:
+# The hypervisor type, one of the values: esx, xen, hyperv, rhevm, libvirt, kubevirt
+  HYPERVISOR_TYPE:
+  # The hostname or ip or other values for hypervisor server, depends on the type
+  HYPERVISOR_SERVER:
+  # Admin username to access hypervisor server
+  HYPERVISOR_USERNAME:
+  # Admin password to access hypervisor server
+  HYPERVISOR_PASSWORD:
+  # The config file to connect hypervisor, only for kubevirt
+  #HYPERVISOR_CONFIG_FILE=
```

Ahhh I missed this the first time! 
```suggestion
  #HYPERVISOR_CONFIG_FILE:
```

---

this was comment out originally, it is not used. but I can change it